packages/material-ui/src/AlertTitle/AlertTitle.test.js

```diff
@@ -1,20 +1,19 @@
 import * as React from 'react';
-import { getClasses, createMount, describeConformance } from 'test/utils';
+import { createMount, describeConformanceV5 } from 'test/utils';
+
 import AlertTitle from './AlertTitle';
+import classes from './alertTitleClasses';
 
 describe('<AlertTitle />', () => {
   const mount = createMount();
-  let classes;
-
-  before(() => {
-    classes = getClasses(<AlertTitle />);
-  });
 
-  describeConformance(<AlertTitle />, () => ({
+  describeConformanceV5(<AlertTitle />, () => ({
     classes,
     inheritComponent: 'div',
     mount,
+    muiName: 'MuiAlertTitle',
+    testVariantProps: { dummy: 'foo' }, // Dont have props to test, but other test must run
```

Comment:
I wonder if we should use some data-test attribute in this case, or if we should actually skip this test cc @oliviertassinari

Reply:
Yes, I wanted to skip it somehow but I can't in this implementation some parts of the tests still needed. Maybe some tests should be separated or allows us to skip.

Reply:
Rename themeComponents -> fullThemeComponents (full suite) and separate those into themeComponentsStyles(only styles), themeComponentsProps (default, variants) 2 smaller suites. fullThemeComponents run both themeComponentsStyles and themeComponentsProps, but you have option run those separately also.

Reply:
```suggestion
```

Reply:
> if we should actually skip this test

This sounds better

Reply:
No it will fail, need also skip: ['componentsProp', 'themeComponents'],

Reply:
Well, I can skip it but styleOverrides won't be tested is it ok for you?

Reply:
I agree we should separate these three tests, and than we can skip only one of them.

Reply:
We should have separate test for each of the theme components parts, one for defaultProps, one for styleOverrides and one for variants.

Reply:
Can not create variants without properties if no properties, no default properties :D

Reply:
Ok, I’ll update the conformance tests on Monday so that we can disable only this one. Good finding @povilass :)